### DISCUSSION TITLE
Question about accuracy of sky views over time

### DISCUSSION THREAD

**Discussion by guyonearth** [Stellarium/stellarium#1429] [Q&A] [answered]
According to Wikipedia, an extremely close great conjunction of Jupiter/Saturn occurred March 1, 1793 BC. Setting this date in Stellarium does not show any such conjunction. The date has to be moved to about January 21, 1792 BC for a conjunction to occur. Am I doing something wrong, or is there some cumulative error happening here? Or is Wikipedia just wrong?

**Comment by gzotti** [comment, marked as answer]
"Wikipedia" is large. Please always add a link to the page in question.

Stellarium User Guide, Appendices F.3 and F.8 provide the answer:
* 1793BC=-1792.
* Julian or Gregorian calendar?

**Reply by guyonearth** [reply]
Stellarium defaults to Julian calendar for that date. Thank you for the answer.
https://en.wikipedia.org/wiki/Great_conjunction
